plugins/postcss-initial/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import type { PluginCreator } from 'postcss';
+import { allProperties, properties } from './properties';
+import { hasExactFallback } from './has-fallback-decl';
+
+/** postcss-initial plugin options */
+export type pluginOptions = {
+	/** Preserve the original notation. default: false */
+	preserve?: boolean,
+};
+
+const IS_INITIAL = /^\s?initial\s?$/i;
+const IS_FONT = /^font$/i;
+const IS_ALL = /^all$/i;
+
+const fontProperties = [
+	'font-family',
+	'font-size',
+	'font-style',
+	'font-variant',
+	'font-weight',
+	'font-stretch',
+	'line-height',
+];
+
+const creator: PluginCreator<pluginOptions> = (opts?: pluginOptions) => {
+	const options: pluginOptions = Object.assign(
+		// Default options
+		{
+			preserve: true,
+		},
+		// Provided options
+		opts,
+	);
+
+	return {
+		postcssPlugin: 'postcss-initial',
+		Declaration(decl) {
+			if (decl.variable) {
+				return;
+			}
+
+			if (!IS_INITIAL.test(decl.value)) {
```

Comment:
`initial` is a css wide keyword.

It must be the only token in a value.

There can be whitespace in front or after but nothing else.
regexp should be sufficient and faster than tokenizing again.